geode-core/src/test/java/org/apache/geode/management/internal/cli/util/ThreePhraseGeneratorTest.java

```diff
@@ -26,10 +27,27 @@
 public class ThreePhraseGeneratorTest {
   @Test
   public void testGenerate() {
-    ThreePhraseGenerator tpg = new ThreePhraseGenerator();
-    String phrase = tpg.generate('-');
+    final ThreePhraseGenerator tpg = new ThreePhraseGenerator();
 
-    assertNotNull(phrase);
-    assertNotEquals("", phrase);
+    final String phrase = tpg.generate('-');
+    assertNotNull("Generated string is not null", phrase);
+    assertNotEquals("Generated string is not empty", "", phrase);
+  }
+
+  @Test
+  public void testGeneratesOnlyLetters() {
+    final ThreePhraseGenerator tpg = new ThreePhraseGenerator();
+
+    // Since there is a random choice of word, try a bunch of times to get coverage.
+    for (int attempt = 1; attempt < 50000; ++attempt) {
+      final String phrase = tpg.generate('x');
+      assertTrue("Generated string has at least five characters", 5 <= phrase.length());
+      for (int index = 0; index < phrase.length(); ++index) {
+        final char c = phrase.charAt(index);
+        assertTrue(
+            "Character in \"" + phrase + "\" at index " + index + ", '" + c + "', is a letter",
```

Comment:
maybe "is _not_ a letter"

Reply:
Nice catch. I'll change all of the messages to indicate the failure instead of the success.